gramps/gen/proxy/filter.py

```diff
@@ -311,102 +317,102 @@ def get_note_from_handle(self, handle):
         else:
             return None
 
-    def get_person_from_gramps_id(self, val):
+    def get_person_from_gramps_id(self, gramps_id: str) -> Person | None:
```

Comment:
This is a different return type than the equivalent method in `DbGeneric` which always returns a `Person`
Is that going to be a problem? Should they be aligned?